cfgov/unprocessed/css/molecules/inset.less

```diff
@@ -46,6 +68,83 @@
         margin-right: unit( @grid_gutter-width / @base-font-size-px, em );
     });
 
+    &__image {
+        float: right;
+
+        div.m-inset_image {
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2678/commits/98cde738538340d61bc985dcc16f0bdb95da36c9.